python/pyspark/sql/connect/expressions.py

```diff
@@ -974,13 +974,3 @@ def to_plan(self, session: "SparkConnectClient") -> proto.Expression:
 
     def __repr__(self) -> str:
         return f"WindowExpression({str(self._windowFunction)}, ({str(self._windowSpec)}))"
-
```

Comment:
I think maybe we can just modify the `to_plan` method in `class DistributedSequenceID`

```
    def to_plan(self, session: "SparkConnectClient") -> proto.Expression:
        unresolved_function = UnresolvedFunction(name="distributed_sequence_id", args=[])
        return unresolved_function.to_plan(session)
```

then we don't need to add `def _distributed_sequence_id ` and change the python tests